project/TastyMiMaFilters.scala

```diff
@@ -63,22 +63,28 @@ object TastyMiMaFilters {
     ProblemMatcher.make(ProblemKind.MissingTermMember, "scala.util.hashing.Hashing.fromFunction"),
     ProblemMatcher.make(ProblemKind.IncompatibleTypeChange, "scala.collection.IterableOnceOps.aggregate"),
     ProblemMatcher.make(ProblemKind.IncompatibleTypeChange, "scala.sys.package.addShutdownHook"),
+    ProblemMatcher.make(ProblemKind.IncompatibleTypeChange, "scala.collection.mutable.HashTable.init"),
+
+    // Probably OK: default parameter
+    ProblemMatcher.make(ProblemKind.MissingTermMember, "scala.*$default$*"),
 
-    // Problems with class constructors
+    // Problem: secondary constructors?
     ProblemMatcher.make(ProblemKind.MissingTermMember, "scala.*.<init>"),
-    ProblemMatcher.make(ProblemKind.IncompatibleTypeChange, "scala.*.<init>"),
+
+    // Problem The symbol scala.*.<init> has a more restrictive visibility qualifier in current version
+    // Only on primitive type final abstract classes
     ProblemMatcher.make(ProblemKind.RestrictedVisibilityChange, "scala.*.<init>"),
 
+    // Problem: constructors have a result type the return unit instead of the class type
+    ProblemMatcher.make(ProblemKind.IncompatibleTypeChange, "scala.*.<init>"), // scala.math.Numeric.CharIsIntegral.<init>; before: (): scala.math.Numeric.CharIsIntegral; after: (): Unit
```

Comment:
That's an OK. In the bytecode constructors don't have result types at all. It must be tasty-query at fault here.